### ISSUE TITLE
Re-introducing stunt objects in the main world

### ISSUE THREAD

**Issue by BeanerLVP** [LVPlayground/playground#720]
I [posted a suggestion](https://forum.sa-mp.nl/thread-35202.html) about this one month ago, but due to the lack of discussion on the forum, I've decided to post it here as well.

What are your opinions about adding stunt objects in all areas excluding the main LV strip, fight club, gang base, gang zones, etc? Now that we have the vehicle keys feature, we can even allow players to freely stunt outside of LV with access to commands to spawn vehicles & keys to flip, repair, jump, etc (unlocked via achievements).

**Comment by SophiaLVP** [comment]
That's a great idea but I do feel that there should be another world for all these things to appear. Whole LV is kind of a war zone right now and I love the way things go around in LV. So another world seeme appropriate for me though. What do you think @BeanerLVP?

**Comment by Specifer** [comment]
No, it is always an enable disable thing. So if someone doesn't want it we could always disable it. And Beaner I would love to spend my time to convert it into json. And if it was already json ,more easier. And I assume it was mapped by Jay? So we could get it done if he is happy to share it?

**Comment by BeanerLVP** [comment]
> 
> 
> That's a great idea but I do feel that there should be another world for all these things to appear. Whole LV is kind of a war zone right now and I love the way things go around in LV. So another world seeme appropriate for me though. What do you think @BeanerLVP?

As I mentioned earlier, we can exclude all the popular DM spots in LV. Also, the gang zones in those areas may have plenty of custom objects of their own, so I agree with you.

LS and SF are barely used for anything other than houses, and we used to have some pretty fun stunts at the LS & SF airports (screenshots in the forum thread). 

When it comes to stunting in other worlds, we already have /jump, but I don't think it's been very popular, and it isolates players to a certain extent. I personally enjoy freeroaming/stunting in the main world over /jump zones. I often find unregistered players driving around and quitting after a while because they most likely lose their vehicle or don't find anything interesting to do. These players are generally the ones that don't speak English and have a hard time exploring our commands and features.

**Comment by EPIC-striker** [comment]
I've been thinking about this and haven't quite got the full scope. 

I think that after joining the server we can give players a few options to choose from, but keeping them in the main world. 
If the player chooses free roam, the player will be moved to LS or SF.
